server.js

```diff
@@ -60,18 +104,16 @@ io.on('connection', (socket) => {
       content: value,
       upvotes: 0
     }
-    messages.push(message)
+    saveMessage('Anon', value)
```

Comment:
We are not sending the correct `id` and `timestamp` value to the clients. `id` will always be `0`. The `messages` array could be removed, it's unused.